openassessment/xblock/test/test_studio.py

```diff
@@ -256,14 +256,16 @@ def test_render_editor_assessment_order(self, xblock):
             'student-training',
             'self-assessment',
             'peer-assessment',
+            'staff-assessment',
         ])
 
         # Change the order (simulates what would happen when the author saves).
         xblock.editor_assessments_order = [
-            'student-training',
             'example-based-assessment',
+            'student-training',
```

Comment:
This was previously an invalid order to be setting, but with example-based not enabled it didn't matter. Now the validation checks are stricter, so I moved it around to make the list valid.